descartes_light/solvers/include/descartes_light/solvers/bgl/impl/bgl_ladder_graph_solver.hpp

```diff
@@ -130,10 +199,44 @@ BuildStatus BGLLadderGraphSolver<FloatType>::buildImpl(
   double duration = std::chrono::duration<double>(Clock::now() - start_time).count();
   CONSOLE_BRIDGE_logDebug("Descartes took %0.4f seconds to build vertices.", duration);
 
+  // Create a zero-value, zero-cost start node and connect it with a zero-cost edge to each node in the first rung
+  {
+    source_ = boost::add_vertex(graph_);
+    auto arr = std::make_shared<State<FloatType>>();
+    graph_[source_].sample = StateSample<FloatType>{ arr, static_cast<FloatType>(0.0) };
+    graph_[source_].rung_idx = -1;
+    for (const VertexDesc<FloatType>& target : ladder_rungs_[0])
```

Comment:
Same here.

Reply:
What needs to be done here?

Reply:
I think clang-tidy will complain about using `const auto&` here.

Reply:
I still don't see where I'm using `const auto&` in this snippet. Regardless, it seems to build fine locally with `clang-tidy` and pass CI, so I'm not sure there is an issue.